auth/server/src/main/java/com/seagates3/policy/PolicyValidator.java

```diff
@@ -397,7 +411,8 @@ ServerResponse checkUnknownElements(JSONObject jsonObject,
     while (keys.hasNext()) {
       String key = keys.next();
       if (!elements.contains(key)) {  // some unknown field found
-        response = responseGenerator.malformedPolicy("Syntax errors in policy");
+        response =
+            responseGenerator.malformedPolicy("Syntax errors in policy.");
```

Comment:
print policy name

Reply:
Policy name is not available here.
But I have added the debug log with policy name included at the time of calling validation from PolicyController#Create.